### PR TITLE
Post template changes for take action boxout block

### PR DESCRIPTION
**Pull request by kirdia**
Removed paragraph splitting in single post template. 
Updated markup in single post template.
Changes made regarding the below PR
[PR presentation layer](https://github.com/greenpeace/planet4-presentation-layer/pull/204)

### CODE REVIEW

File: templates/single.twig [hunk 1]
```diff
@@ -36,29 +36,11 @@
 		<!-- Post Block Start -->
 		<div class="container">
 			<div class="post-content">
+				{% if ( post.take_action_boxout ) %}
+					{{ post.take_action_boxout|shortcodes|raw }}
```
Comment (angtheod) [inline templates/single.twig]: Do we need escaping here together with the |raw filter? Or is it already escaped?

Comment (kirdia) [reply templates/single.twig]: No its is not escaped. I think we will need escaping here. 
Good catch, I will update the code.

Comment (kirdia) [reply templates/single.twig]: Actually escaping here will remove style attribute from top level element in the take action action boxout block (no background image).
I will add escaping in the take action boxout block itself, so I think we are good to go with this.

Comment (angtheod) [reply templates/single.twig]: ok

**Pull request merged**